README.md

```diff
@@ -150,7 +150,7 @@ service<http:Service> LocalShop bind LocalShopEP {
 Navigate to `pass-through` and run the following command in the command line to start `passthrough.bal`.
 
 ```bash
-   $ ballerina run passthrough.bal
+   $ ballerina run passthrough
```

Comment:
need to update the comment "Navigate to `pass-through` and run the following command in the command line to start `passthrough.bal`." too?

Reply:
Maybe not because this command also will start the services in `passthrough.bal` file

Reply:
Then shouldn't the comment be
"Navigate to `pass-through-messaging/guide` and run the following command in the command line to start  the services in `passthrough.bal`." ?

Reply:
Fixed